js/components/changesetNotice.js

```diff
@@ -67,6 +67,7 @@ const create_unified_changeset_notice = ({
     cancel_text = gettext('Cancel'),
     context = null,
     notice_type = 'notice', // possible values: error (danger/red), info (blue), success (green), notice (warning/yellow)
+    modal = false, // notice will act like a modal with the rest of the page inactive until the notice is responded to
```

Comment:
maybe "if set to true, notice will act..."?

Reply:
Yes, that makes sense.